THIRD-PARTY-LICENSES.properties

```diff
@@ -69,4 +69,5 @@ com.readytalk--metrics-statsd-common--4.2.0=WDL License https://github.com/openw
 org.antlr--antlr-runtime--3.4=The BSD License (https://www.antlr.org/license.html)
 dom4j--dom4j--1.6.1=https://github.com/dom4j/dom4j/blob/master/LICENSE
 io.netty--netty-tcnative-classes--2.0.46.Final=https://github.com/netty/netty-tcnative/blob/main/LICENSE.txt
+org.w3id.cwl.sdk--cwljava--1.0.0=https://www.apache.org/licenses/LICENSE-2.0.txt
```

Comment:
Looking at the formatting below for the "normal" dependencies, wonder if this should be:
`org.w3id.cwl.sdk--cwljava--1.0.0=(Apache License, Version 2.0)`

Reply:
Alternatively, maybe you can make a PR to cwljava having them add a license file then we can link to it specifically rather than use the the generic link. Think that may be a better to give them a heads up that we're using it and/or give them a shout-out.